rust/pkg/cardano_multiplatform_lib.js.flow

```diff
@@ -348,6 +348,23 @@ declare export var StakeCredKind: {|
   +Script: 1, // 1
 |};
 
+/**
+ */
+
+declare export var AddressHeader: {|
+  +BasePaymentKeyStakeKey: 0, // 0
```

Comment:
The values generated by Flow here are actually wrong (Typescript is correct)

Just another reason to drop Flow :/

Reply:
with wasm_bindgen/flow you should never be using the actual integer value of an enum. it doesn't matter what you do it always sets them to 0, 1, ... etc. I had assumed this was just a wasm_bindgen thing and that typescript was affected as well (since flow is generated from that) but that's good to know that typescript retains the explicit enum values.